librz/include/rz_util/rz_num.h

```diff
@@ -23,6 +23,7 @@ typedef enum {
 	RNCPLUS = '+',
 	RNCMINUS = '-',
 	RNCMUL = '*',
+	RNCEXP = '**',
```

Comment:
I think it is not really needed to put this here with any `=... `. In particular, this seems wrong as you can just assign one character to the enum value.

Reply:
Yes, true. Then I am changing this to capital `E` is that okay?